packages/playwright/src/reporters/base.ts

```diff
@@ -547,3 +547,26 @@ function fitToWidth(line: string, width: number, prefix?: string): string {
 function belongsToNodeModules(file: string) {
   return file.includes(`${path.sep}node_modules${path.sep}`);
 }
+
+function resolveFromEnv(name: string): string | undefined {
+  const value = process.env[name];
+  if (value)
+    return path.resolve(process.cwd(), value);
+  return undefined;
+}
+
+export function resolveOutputFile(reporterName: string, options: { configDir: string, outputFile?: string }): string|undefined {
```

Comment:
Can we reuse this in `blob.ts`?

Reply:
There are too many differences like default dir, output name resolution, fileName configurable parameter, default file name vs printing to the stdout, so the code is simpler if they are kept separate.